CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+
+#### Python Virtual Environment setup
+
+For python it is best to setup a virtual environment dedicated to mentat development to avoid any conflicts with other projects. This section is not meant to be comprehensive in setting up python virtual environments. There are good tutorials on how to do that such as [Python Virtual Environments](https://www.arch.jhu.edu/python-virtual-environments/).  the notes below are supplemental to help with Mentat development.
+
+##### Conda (anaconda) Specific Notes
+
+Since not all the requirements are available in the conda repositories, it will be necessary to use pip to install the requirements. In order to avoid conflicts between installation using conda and pip you should add the following to your `.condarc` file (the location of the file varies based on the OS - but typically is in the users home directory):
+
+```
+pip_interop_enabled: true
+```
+
+## Commit Message Guidelines
+
+The guidelines below are for commit messages. While currently not enforced, eventually we will enforce the guidelines and use them for the [CHANGELOG.md][changelog-uri] generation and we strongly encourage following the guidelines.
+
+```text
+<TYPE>(<SCOPE>): imperative subject in present tense less than 80 characters
+NOTES:
+Add any additional notes in imperative present tense.
+BREAKING CHANGE:
+Describe the previous behavior in contrast to the new behavior and how it might effect consumers.
+```
+
+Allows types are:
+
+* **feat**: represents a new feature
+* **fix**: represents a bug fix
+* **build**: changes to the gitlab pipeline or build scripts
+* **refactor**: refactor code without expecting any behavioral change
+* **test**: updates to test code only
+* **perf**: updates for performance only (should include a benchmark for verification)
+* **docs**: update to documentation only
+* **style**: changes to style only (reformatting, whitespace changes, etc)
+
+The `(<SCOPE>)` argument should only be included if the change is isolated to a single module. This should be the name
+of the module that was modified.
+
+Only include a BREAKING CHANGE footer if necessary. Only include a NOTES body if it adds appropriate value.
+
+The header line **SHOULD** be in all lowercase characters and exclude any special characters other than the semi-colon
+and the parentheses around the scope (if included).
+
+Example commit message:
+
+```text
+docs: add documentation on contributing.
+* initial creation of the contributing documentation.
+NOTES:
+Contirbution follows the usual Github open source contribution mechanism. However, there are some customizations specific to Mentat. The general approach, where to find more information and Mentat specific guidelines are documented here.
```

Comment:
Typo. Also I'd prefer to omit lines 77 and 78 entirely.

Reply:
Made the changes